facebookresearch/LIGHT
# Remaining database cleanup, tests now passing.

**Patch description**


**Testing steps**
```bash
>> pytest light
...............................................sssssssssssss.........ssssssssss..s..s..s..s....................................................................................................... [ 88%]
.........................                                                                                                                                                                          [100%]
============================================================================================ warnings summary ============================================================================================
/Users/jju/miniconda3/envs/py38/lib/python3.8/site-packages/numexpr/expressions.py:21
/Users/jju/miniconda3/envs/py38/lib/python3.8/site-packages/numexpr/expressions.py:21
  /Users/jju/miniconda3/envs/py38/lib/python3.8/site-packages/numexpr/expressions.py:21: DeprecationWarning: distutils Version classes are deprecated. Use packaging.version instead.
    _np_version_forbids_neg_powint = LooseVersion(numpy.__version__) >= LooseVersion('1.12.0b1')

/Users/jju/miniconda3/envs/py38/lib/python3.8/site-packages/scipy/special/orthogonal.py:79
/Users/jju/miniconda3/envs/py38/lib/python3.8/site-packages/scipy/special/orthogonal.py:79
  /Users/jju/miniconda3/envs/py38/lib/python3.8/site-packages/scipy/special/orthogonal.py:79: DeprecationWarning: `np.int` is a deprecated alias for the builtin `int`. To silence this warning, use `int` by itself. Doing this will not modify any behavior and is safe. When replacing `np.int`, you may wish to use e.g. `np.int64` or `np.int32` to specify the precision. If you wish to review your current use, check the release note link for additional information.
  Deprecated in NumPy 1.20; for more details and guidance: https://numpy.org/devdocs/release/1.20.0-notes.html#deprecations
    from numpy import (exp, inf, pi, sqrt, floor, sin, cos, around, int,

light/world/tests/test_agent_death.py::TestInteractionLoggers::test_run
  /Users/jju/miniconda3/envs/py38/lib/python3.8/asyncio/events.py:81: RuntimeWarning: coroutine 'BattleRoyaleSoul.observe_event' was never awaited
    self._context.run(self._callback, *self._args)

-- Docs: https://docs.pytest.org/en/latest/warnings.html
======================================================================================== short test summary info =========================================================================================
SKIPPED [8] light/graph/builders/tests/test_StarSpaceBuilder.py: Need to update starspace builders to use Model Pool
SKIPPED [5] light/graph/builders/tests/test_StarspaceNeighborBuilder.py: Need to update starspace builders to use Model Pool
SKIPPED [7] light/graph/builders/tests/test_user.py: Need to update starspace builders to use Model Pool
SKIPPED [3] /Users/jju/miniconda3/envs/py38/lib/python3.8/site-packages/_pytest/unittest.py:258: Skip GraphEventTests tests, it's a base class
SKIPPED [1] light/graph/tests/test_events.py:230: No valid events for <class 'light.graph.events.graph_events.SayEvent'>
SKIPPED [1] light/graph/tests/test_events.py:230: No valid events for <class 'light.graph.events.graph_events.ShoutEvent'>
SKIPPED [1] light/graph/tests/test_events.py:230: No valid events for <class 'light.graph.events.graph_events.WhisperEvent'>
SKIPPED [1] light/graph/tests/test_events.py:230: No valid events for <class 'light.graph.events.graph_events.TellEvent'>
192 passed, 27 skipped, 5 warnings in 14.29s
```
